packages/parchment/lib/src/document/embeds.dart

```diff
@@ -93,14 +89,9 @@ class SpanEmbed extends EmbeddableObject {
 ///
 /// Examples of block embeds include horizontal rule, an image or a map view.
 ///
-/// There are two built-en embed types supported by Parchment documents, however
+/// There are two built-in embed types supported by Parchment documents, however
 /// the document model itself does not make any assumptions about the types
 /// of embedded objects and allows users to define their own types.
-///
```

Comment:
Is there a reason for removing this comment?

Reply:
Since it's not the case anymore. Fleather doesn't suppose anything about horizontal rule or image. You can redefine them easily. Am I wrong? @amantoux

Reply:
`FleatherEditor` still implements a defaultEmbedBuilder that supports 'hr", doesn't it?

Reply:
It doesn't implement it. It uses a default `EmbedBuilder` which is simply overridable as a constructor parameter. @amantoux